dhcp/install.sh

```diff
@@ -2,12 +2,33 @@
 
 source /vagrant/settings.sh
 
-#installation
+# installation
 apt --yes update
 apt --yes --install-recommends install dnsmasq
+export DEBIAN_FRONTEND=noninteractive
+apt -yq --yes install iptables-persistent
```

Comment:
Here `-yq` is not needed.

Reply:
`-yq` flag is required to handle `DEBIAN_FRONTEND` variable.

Reply:
> -yq flag is required to handle DEBIAN_FRONTEND variable.

Are you sure about this? Where did you find this information?

Reply:
I can share the source - https://serverfault.com/a/227194

Reply:
Look up in the man page the meaning of `-yq` and then remove it.
You have spent me 1 our only for this, and it is not normal that I spend in the project as much time as you do.